cluster/examples/kubernetes/ceph/create-external-cluster-resources.py

```diff
@@ -0,0 +1,283 @@
+            raise ExecutionFailureException(
+                "'auth get-or-create client.csi-cephfs-provisioner' command failed")
+        if len(json_out) > 0:
+            return str(json_out[0]['key'])
+        raise ExecutionFailureException(
+            "Failed to get/create 'client.csi-cephfs-provisioner'")
+
+    def create_cephCSIKeyring_cephFSNode(self):
+        cmd_json = {"prefix": "auth get-or-create",
+                    "entity": "client.csi-cephfs-node",
+                    "caps": [
+                        "mon", "allow r",
+                              "mgr", "allow rw",
+                              "osd", "allow rw tag cephfs *=*",
+                              "mds", "allow rw"],
+                    "format": "json"}
+        ret_val, json_out = self._common_cmd_json_gen(cmd_json)
+        # if there is an unsuccessful attempt,
+        if ret_val != 0:
+            raise ExecutionFailureException(
+                "'auth get-or-create client.csi-cephfs-node' command failed")
+        if len(json_out) > 0:
+            return str(json_out[0]['key'])
+        raise ExecutionFailureException(
+            "Failed to get/create 'client.csi-cephfs-node' key")
+
+    def create_cephCSIKeyring_RBDProvisioner(self):
+        cmd_json = {"prefix": "auth get-or-create",
+                    "entity": "client.csi-rbd-provisioner",
+                    "caps": ["mon", "profile rbd",
+                             "mgr", "allow rw",
+                             "osd", "profile rbd"],
+                    "format": "json"}
+        ret_val, json_out = self._common_cmd_json_gen(cmd_json)
+        # if there is an unsuccessful attempt,
+        if ret_val != 0:
+            raise ExecutionFailureException(
+                "'auth get-or-create client.csi-rbd-provisioner' command failed")
+        if len(json_out) > 0:
+            return str(json_out[0]['key'])
+        raise ExecutionFailureException(
+            "Failed to get/create 'client.csi-rbd-provisioner' key")
+
+    def create_cephCSIKeyring_RBDNode(self):
+        cmd_json = {"prefix": "auth get-or-create",
+                    "entity": "client.csi-rbd-node",
+                    "caps": ["mon", "profile rbd",
+                             "osd", "profile rbd"],
+                    "format": "json"}
+        ret_val, json_out = self._common_cmd_json_gen(cmd_json)
+        # if there is an unsuccessful attempt,
+        if ret_val != 0:
+            raise ExecutionFailureException(
+                "'auth get-or-create client.csi-rbd-node' command failed")
+        if len(json_out) > 0:
+            return str(json_out[0]['key'])
+        raise ExecutionFailureException(
+            "Failed to get/create 'client.csi-rbd-node' key")
+
+    def create_checkerKey(self):
+        cmd_json = {"prefix": "auth get-or-create",
+                    "entity": self.run_as_user,
+                    "caps": ["mon", "allow r, allow command quorum_status"],
```

Comment:
Please add the following caps to it: https://github.com/rook/rook/pull/5372/files#diff-92ddf7a4461086b6845bc37a93dded0fR35

Reply:
Made required changes...